tests/rustdoc-ui/error-in-impl-trait/infinite-recursive-type-2.rs

```diff
@@ -0,0 +1,3 @@
+pub enum E { //~ ERROR
+    V(E),
+}
```

Comment:
I meant exactly the same as the original test, but with both types public:
```suggestion
pub fn f() -> impl Sized {
    pub enum E {
        V(E),
    }

    unimplemented!()
}
```

Reply:
Oh I see. Updating then.